src/components/Clicker.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+class Clicker extends React.Component {
+    
+        state = { count: 0 };
```

Comment:
the state object should contain all the other information ex;

`state ={ count : 0, increase : 1}`